sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -171,6 +172,30 @@ class SparkSessionExtensionSuite extends SparkFunSuite {
     }
   }
 
+  test("reset column vectors") {
```

Comment:
Thank you for your first contribution, @rongma1997 .
For the bug fix test case, we need `SPARK-29490` prefix for the test case name.